### DISCUSSION TITLE
Fahrassen durch CP erstellen lassen

### DISCUSSION THREAD

**Discussion by JanBrockmann** [Courseplay/Courseplay_FS22#2451] [Ideas]
Hallo CP-Team,

da Courseplay die Fahrspuren berechnen kann, wäre es ein nettes Feature auf diesen Spuren optional die Feldfrucht zu entfernen. Man hätte dann ohen größeren Aufwand in den Felder saubere Fahrgassen. Meines Erachtens nach würde es das Spiel sehr aufwerten und die Fahrgassen mit ProSeed zu erstellen ist ja nicht jedermanns Sache. :-)

**Comment by Tensuko** [comment]
#2362
Mehr wird es da nicht geben von uns, wenn überhaupt, wie auch schon oft genug erwähnt.
Die Fahrgassen werde dann auch keine Nachfolgenden Funktionen haben wie für Feldspritzen etc.
Geht halt nicht einfach mal so.
